server/routes/api/import_detection/place_detection.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Place Detection."""
+
+from typing import Dict, List, Optional, Tuple
+from routes.api.import_detection.detection_types import TypeProperty
+from routes.api.import_detection.country_state_detector import CountryStateDetector
+from routes.api.import_detection.place_detector_abstract import PlaceDetectorInterface
+
+_MIN_HIGH_CONF_DETECT: float = 0.4
+"""Returns the list of supported Place detectors."""
+PLACE_DETECTORS: Tuple[PlaceDetectorInterface, ...] = (
```

Comment:
nit:  Can this line be a comment instead of a string that just says "List of supported place detectors"?  Also, vertical space after _MIN_HIGH_CONF

Reply:
Thanks! Fixed.